lib/xchain/provider/fetch.go

```diff
@@ -15,26 +16,18 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 )
 
-func getCurrentFinalisedBlockHeader(ctx context.Context, rpcClient *ethclient.Client) (*types.Header, error) {
-	// skip ethCLient and call the function directly as the "finalized" tag is not supported
-	// by ethClient. This call will return the last finalized block.
-	// var finalisedHeader types.Header
-	// params := []string{"latest", "false"}
-	// err := rpcClient.Client().CallContext(ctx, &finalisedHeader, "eth_getBlockByNumber", params)
-	// if err != nil {
-	//	 return nil, errors.Wrap(err, "could not get finalized block")
-	// }
-
-	// TODO(corver): Support different finalized methods (to be added to netconf).
-	//  The only chain we support at this point is anvil, it doesn't support "finalized", so just use "latest" for now.
-	height, err := rpcClient.BlockNumber(ctx)
+func getCurrentFinalisedBlockHeader(
```

Comment:
Please, [don't use `get` as function or method prefixes](https://google.github.io/styleguide/go/decisions#getters). It is neither go-idiomatic, nor necessary.